src/modules/admin/components/services/ServiceTypeDetailPage.tsx

```diff
@@ -79,24 +80,44 @@ const ServiceTypeDetailPage = () => {
         }
       />
       {serviceType && (
-        <UpdateServiceTypeDialog
+        <ServiceTypeDialog
           serviceType={serviceType}
           dialogOpen={updateDialogOpen}
           closeDialog={() => setUpdateDialogOpen(!updateDialogOpen)}
         />
       )}
       <Stack gap={2}>
+        <Alert color='info'>
```

Comment:
```suggestion
        <Alert severity='info'>
```

To get the "info" icon instead of the success icon

Reply:
Maybe also only show this alert if `serviceType.formDefinitions.length === 0`? "To enable this service in a project..." feels like a call-to-action that may not be appropriate for a service type that's already enabled.